Source/UIKit/UIPageControl.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  UIPageControl.swift
+//  Rex
+//
+//  Created by Torsten Curdt on 7/24/16.
+//  Copyright (c) 2015 Torsten Curdt. All rights reserved.
+//
+
+import ReactiveCocoa
+import UIKit
+
+extension UIPageControl {
+
+    /// Wraps a page control's `numberOfPages` value in a bindable property.
+    public var rex_numberOfPages: MutableProperty<Int> {
+        return associatedProperty(self, key: &numberOfPagesKey, initial: { $0.numberOfPages }, setter: { $0.numberOfPages = $1 })
+    }
+
+    /// Wraps a page control's `currentPage` value in a bindable property.
+    public var rex_currentPage: MutableProperty<Int> {
+        return associatedProperty(self, key: &currentPageKey, initial: { $0.currentPage }, setter: { $0.currentPage = $1 })
```

Comment:
It will keep `currentPage` and `rex_currentPage` in sync. Right now we can control `UIPageControl`'s `currentPage` but if the user interacts with the page control changing to a certain page manually, triggering a `UIControlEventValueChanged` event, our bindable property will not be updated with the new index.